src/components/CalendarDay.jsx

```diff
@@ -94,14 +85,13 @@ class CalendarDay extends React.PureComponent {
   render() {
     const {
       day,
-      ariaLabelFormat,
       daySize,
       isOutsideDay,
       modifiers,
       renderDayContents,
       tabIndex,
       styles,
-      phrases,
+      ariaLabel,
```

Comment:
I'm not really sure what is considered part of the public API here and if this is a breaking change or not.

Reply:
This is a breaking change. This would definitely break the pdp calendar

Reply:
Removal of any prop is generally a breaking change.

Reply:
Could we keep both, and ignore the old one when the new one is present?

Reply:
Maybe. Alternatively, I think I might try leaving the phrases prop here and instead pass down `focusedInput` so we don't have to change `phrases` every time. I want to think through how we might get this optimization working for more days without breaking accessibility before we choose a direction to go in here.

Reply:
I think in order to optimize most dates, we will need to avoid changing `ariaLabel` when `focusedInput` changes. I think in order to do that, we can render at a higher level (`CalendarMonthGrid` maybe) a hidden node that manages the phrase for the focused date, and pass down an id to CalendarDay for `aria-labelledby`. Still need to think through `CustomizableCalendarDay` I think.

Reply:
I've added a commit to this PR that takes the `aria-labelledby` approach.